### ISSUE TITLE
West Manga URL Change

### ISSUE THREAD

**Issue by Arisa43fps** [keiyoushi/tachiyomi-extensions-source#76]
### Source information

West Manga 1.4.29

### Source language

Indonesia

### Source new URL

https://westmanga.fun/

### Other details

The old url has been blocked by kominfo, so it can't be accessed by most isp in my country Indonesia. The old URL can still be visited but must use a VPN with an overseas connection.

### Acknowledgements

- [X] I have updated all installed extensions.
- [X] I have opened WebView and checked that the source URL is not updated yet.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.
